alwaysontop/main/utils.js

```diff
@@ -1,129 +1,72 @@
-const os = require('os');
+const store = require('electron-store');
```

Comment:
This does not work for me, I always (on Linux and Mac) get the following error:

![aot-electron-store](https://user-images.githubusercontent.com/1392875/139730208-1990bf0c-5925-4218-b802-60b0d1f307aa.png)

According to the docs of electron-store, it should be something like:

```
const Store = require('electron-store');

const store = new Store();
```

I'll push this as a fixup commit and test.